app/index.py

```diff
@@ -204,7 +204,7 @@ def bulk_index(self, actions: Iterable[dict]):
         successes = 0
 
         for ok, _ in helpers.streaming_bulk(
-            client=self.opns, index=self.index_name, actions=actions
+            client=self.opns, index=self.index_name, actions=actions, timeout="30s"
```

Comment:
"magic" number here - is this worth pulling out into config at some stage?

Reply:
done!